uqcsbot/whatweekisit.py

```diff
@@ -0,0 +1,162 @@
+import requests
+from collections import namedtuple
+from typing import NamedTuple, List, Tuple, Optional
```

Comment:
Given that the discord bot uses python3.9, try to use `list` instead of `List` as this is considered deprecated - same goes for `tuple` instead of `Tuple`

Reply:
I agree but I'd prefer the codebase to be consistent with regards to typing (presently, the rest of the codebase uses `List`, `Tuple`, etc.) -- I think it'd be better to leave it as `List`/`Tuple` for now, and if the maintainers of the codebase decide that they don't want to support backwards compatibility with previous versions of Python and that the overhead of changing what's effectively different syntactic sugar is worth it, they can make an overarching pull request fixing all instances of it in the repo.

Reply:
i aint reading all that but good for you or sorry that it happened

Reply:
also i'm expecting credit for original idea

Reply:
i will consider this matter resolved and you may expect to hear from my lawyer soon

Reply:
I know you literally cannot afford a lawyer with your current weekly salary and rent payments